core/src/main/java/com/cloud/agent/api/routing/SetMonitorServiceCommand.java

```diff
@@ -29,21 +32,34 @@
  * how to access the components inside the command.
  */
 public class SetMonitorServiceCommand extends NetworkElementCommand {
-    MonitorServiceTO[] services;
+    public static final String ROUTER_MONITORING_ENABLED = "router.monitor.enabled";
+    public static final String ROUTER_HEALTH_CHECKS_ENABLED = "router.health.checks.enabled";
+    public static final String ROUTER_HEALTH_CHECKS_BASIC_INTERVAL = "router.health.checks.basic.interval";
+    public static final String ROUTER_HEALTH_CHECKS_ADVANCED_INTERVAL = "router.health.checks.advanced.interval";
+    public static final String ROUTER_HEALTH_CHECKS_EXCLUDED = "router.health.checks.excluded";
+
+    private MonitorServiceTO[] services;
+    private Map<String, String> healthChecksConfig;
+    private boolean reconfigureAfterUpdate;
+    private boolean deleteFromProcessedCache;
 
-    protected SetMonitorServiceCommand() {
+    public SetMonitorServiceCommand() {
```

Comment:
nit: didn't find its public usage or maybe I missed

Reply:
Removing back to protected. I think public usage went away during CR comments.